server/models/remote_provider.go

```diff
@@ -732,6 +732,7 @@ func (l *RemoteProvider) SaveK8sContext(token string, k8sContext K8sContext) (co
 		Kind:    "kubernetes",
 		Type:    "platform",
 		SubType: "orchestrator",
+		Name: k8sContext.Name,
```

Comment:
Do we know if this section of code strictly and only deals with Kubernetes connections or do we need to also consider this change in context of different connection types?

Reply:
i guess from the values above Kind: "kubernetes"  . this piece is specifically for kuberntes